packages/modules/common/store/_chargepoint.py

```diff
@@ -1,6 +1,22 @@
 from modules.common.component_state import ChargepointState
 from modules.common.store import ValueStore
+from modules.common.store._api import LoggingValueStore
 from modules.common.store._broker import pub_to_broker
+from modules.common.store.ramdisk import files
+from helpermodules import compatibility
+
+
+class ChargepointValueStoreRamdisk(ValueStore[ChargepointState]):
+    def __init__(self, cp_id: int):
+        self.num = cp_id
+
+    def set(self, cp_state: ChargepointState):
+        files.charge_points[self.num].is_charging.write(cp_state.charge_state)
+        files.charge_points[self.num].voltages.write(cp_state.voltages)
+        files.charge_points[self.num].currents.write(cp_state.currents)
+        files.charge_points[self.num].energy.write(cp_state.imported)
+        files.charge_points[self.num].is_plugged.write(cp_state.plug_state)
+        files.charge_points[self.num].power.write(cp_state.power)
```

Comment:
Der Nerd in mir will folgendes vorschlagen:
```suggestion
        charge_point = files.charge_points[self.num]

        charge_point.is_charging.write(cp_state.charge_state)
        charge_point.voltages.write(cp_state.voltages)
        charge_point.currents.write(cp_state.currents)
        charge_point.energy.write(cp_state.imported)
        charge_point.is_plugged.write(cp_state.plug_state)
        charge_point.power.write(cp_state.power)
```
Ist dann nurnoch 1x statt 6x ein Zugriff auf `files.charge_points[self.num]` und spart entsprechend Objekterzeugungen ein. Der Performancevorteil dürfte allerdings kaum messbar ausfallen...